signer/cosigner_nonce_cache_test.go

```diff
@@ -45,6 +68,9 @@ func TestNonceCacheDemand(t *testing.T) {
 
 	cancel()
 
-	target := int(nonceCache.noncesPerMinute*.01) + 10
+	target := int(nonceCache.noncesPerMinute*.01) + int(nonceCache.noncesPerMinute/30) + 100
```

Comment:
Can you put a comment for this logic? E.g. how does `int(nonceCache.noncesPerMinute*.01)` and `100` map to the `target` in the `reconcile` function?. Is this check for ensuring the cache doesn't grow unbounded?

Reply:
It was supposed to be the same target as in the `reconcile` method. I had changed the `+ 10` to `+ 100` then back, but forgot to update the test. I pulled the `target` out to a method so it can be called in the test instead.